### ISSUE TITLE
mariadb: unsupported type "inet4"

### ISSUE THREAD

**Issue by shadow3x3x3** [ariga/atlas-provider-gorm#46]
Hi,

We try to `migrate diff` this model with type `INET4`.

```go
type BlockIP struct {
	IP string `gorm:"primaryKey;type:INET4"`

	CreatedAt time.Time `gorm:"column:created_at;not null"`
}
```

The error occurs 

```log
Error: calculate reverse for drop table "block_ips": create table "block_ips": format type for column "ip": unsupported type "inet4"
```

We use mariadb that version supports `INET4` type.

**Comment by giautm** [comment]
@luantranminh, please take a look at this issue, I think we need a small patch on @ariga/atlas to support this type for MariaDB

**Comment by luantranminh** [comment]
yah, I'm working on this one.

**Comment by luantranminh** [comment]
hey @shadow3x3x3 , We just released Atlas [v0.23.0](https://github.com/ariga/atlas/releases/tag/v0.23.0), which now supports `inet4` and `inet6`.
Upgrade your Atlas tool and run the migration again, and it will work.

```bash
curl -sSf https://atlasgo.sh | sh
```
